src/xdg-autostart-generator/xdg-autostart-service.c

```diff
@@ -80,13 +81,10 @@ static int xdg_config_parse_bool(
         assert(rvalue);
         assert(data);
 
-        if (streq(rvalue, "true"))
-                *b = true;
-        else if (streq(rvalue, "false"))
-                *b = false;
-        else
+        int r = parse_boolean(rvalue);
```

Comment:
May not explicitly documented, but `int r` is a kind of special variable we usually store the return value. Please declare it at the very beginning of the function.